### PR TITLE
Tech report re-write

### PR DESCRIPTION
**Pull request by KVTaniguchi**
@ everyone at Wire - My sincere apologies for letting this drag on for so long and gratitude for allowing me to contribute to this project.  I decided to generate a new branch and PR b/c the previous one had fallen too far behind the ```development``` branch and become unmaintainable.
Previous PR: https://github.com/wireapp/wire-ios/pull/119
- added file from previous PR
- removed legacy objc file
- updated brigdging header to use appDel logging

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Settings/SettingsTechnicalReportViewController.swift [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        switch indexPath.section {
+        case TechnicalReportSection.Options.rawValue:
+            return indexPath.row == 0 ? includedVoiceLogCell : sendReportCell
+        default:
+            let cell = tableView.dequeueReusableCellWithIdentifier(technicalReportReuseIdentifier, forIndexPath: indexPath)
+            let TechnicalReport = lastCallSessionReports[indexPath.row]
+            cell.detailTextLabel?.text = TechnicalReport[SettingsTechnicalReportViewController.technicalReportData]
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Settings/SettingsTechnicalReportViewController.swift]: The cells with tech report info are missing titles:
was set like `cell.textLabel.text = technicalReport[TechnicalReportTitle];`

I think this should be the final change, sorry that we did not noticed that during the initial review.

New:
img_4497
Old:
img_4496

**Comment by mikeger** [comment]
@KVTaniguchi It happened to be that we AGAIN updated the project file, and settings too. If you would like we can also fix the conflicts from your branch (you need to grant extra privileges).

Right now it seems that conflict is minor. Also I hope that you enjoy the new settings UI!

**Comment by KVTaniguchi** [comment]
updated

**Comment by mikeger** [comment]
@KVTaniguchi thanks a lot for your patience and commitment! I'll update the screen for the new design (white on black).

**Pull request merged**